test/integration/websocket_integration_test.cc

```diff
@@ -538,8 +490,6 @@ TEST_P(WebsocketIntegrationTest,
     return;
   }
 
-  config_helper_.addRuntimeOverride(
```

Comment:
You can remove this entire test, as it is now exactly the same as the one above it.

Reply:
Thanks, removed.